cni/scripts/cleanupnetwork.ps1

```diff
@@ -0,0 +1,73 @@
+ # ./cleanupnetwork.ps1 -CniName azure
+ param (
+    [Parameter(Mandatory=$true)][string]$CniName
+ )
+
+Invoke-WebRequest -Uri https://raw.githubusercontent.com/microsoft/SDN/master/Kubernetes/windows/hns.psm1 -OutFile "c:\hns.psm1" -UseBasicParsing
+
+$global:NetworkMode = "L2Bridge"
+$global:HNSModule = "c:\hns.psm1"
+
+ipmo $global:HNSModule
+
+$networkname = $global:NetworkMode.ToLower()
```

Comment:
can we assume default is "azure" even if this parameter not specified?

Reply:
this script is for multitenancy..so it will create network with azure_vlan.. is there a way to get networks with prefix match and delete it?